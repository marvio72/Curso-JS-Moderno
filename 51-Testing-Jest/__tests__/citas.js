import Citas from '../js/classes/Citas';

describe('Probar la clase de citas', () => {
  const citas = new Citas();

  test('Agregar una nueva cita', () => {
    const citaObj = {
      mascota: 'Roco',
      propietario: 'Marco',
      telefono: '33434321',
      fecha: '10-12-2020',
      hora: '10:30',
      sintomas: 'Solo duerme',
    };

    citaObj.id = Date.now();

    citas.agregarCita(citaObj);

    // Prueba
    expect(citas).toMatchSnapshot();
  });
});
