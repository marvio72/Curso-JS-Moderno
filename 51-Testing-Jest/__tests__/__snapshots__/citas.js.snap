// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probar la clase de citas Agregar una nueva cita 1`] = `
Citas {
  "citas": Array [
    Object {
      "fecha": "10-12-2020",
      "hora": "10:30",
      "id": 1611806508078,
      "mascota": "Roco",
      "propietario": "Marco",
      "sintomas": "Solo duerme",
      "telefono": "33434321",
    },
  ],
}
`;
